bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -396,6 +424,32 @@
             commandId="org.plugin.espressif.commands.eraseFlash">
       </handler>
    </extension>
+   <extension
+         point="org.eclipse.ui.menus">
+      <menuContribution
+            locationURI="menu:org.eclipse.ui.main.menu?after=additions">
+         <menu
+               id="org.plugin.espressif.menus.espressifMenu"
+               label="%menu.label.1"
+               mnemonic="%menu.mnemonic">
+            <command
+                  commandId="org.plugin.espressif.commands.eraseFlash"
+                  icon="icons/Erase Flash.png"
+                  id="org.plugin.espressif.menus.eraseFlashCommand"
+                  label="%command.label.5">
+            </command>
+            <separator
+                  name="com.espressif.idf.bundles.separator1"
+                  visible="true">
+            </separator>
+            <command
+                  commandId="com.espressif.idf.bundles.installIdfComponents"
+                  id="com.espressif.idf.bundles.installIdfComponentsCommand"
+                  label="%command.name.10">
+            </command>
+         </menu>
+      </menuContribution>
+   </extension>
```

Comment:
Looks like you have merged master to your branch instead could you do rebase on top of the master so that you don't get the merged changes as part of your branch

Reply:
Ok will do it from now on cannot rebase as the branch is now in sync with master

Reply:
Hi @alirana01 That's going to complicate the branch hierarchy. I would request you to change this. Use "chery-pick" approach.